src/backend/launcher.ts

```diff
@@ -749,7 +758,7 @@ export async function verifyWinePrefix(
     return { res: { stdout: '', stderr: '' }, updated: false }
   }
 
-  if (!existsSync(winePrefix)) {
+  if (!existsSync(winePrefix) && wineVersion.type !== 'proton') {
```

Comment:
This change is the one preventing proton without UMU from creating the prefix when needed, so maybe:

```suggestion
  if (!existsSync(winePrefix) && !umuSupported(wineVersion.type)) {
```

so it still creates the prefix folder for proton when umu is disabled